examples/14_react-tweet/src/templates/home-page.tsx

```diff
@@ -0,0 +1,26 @@
+import { Counter } from '../components/counter.js';
+import { Tweet } from 'react-tweet';
+
+export const HomePage = async () => {
+  const data = await getData();
+
+  return (
+    <div>
+      <title>{data.title}</title>
+      <h1 className="text-4xl font-bold tracking-tight">{data.headline}</h1>
+      <p>{data.body}</p>
+      <Tweet id="1735308967880823082" />
```

Comment:
![image](https://github.com/dai-shi/waku/assets/37929992/86aae27f-35ae-479d-931f-e8b6e66eb70f)

That's legacy here 😄